.github/workflows/benchmarks-nightly.yml

```diff
@@ -0,0 +1,38 @@
+name: Compute Benchmarks Nightly
```

Comment:
nit: could add badge for this nightly workflow

Reply:
done